### DISCUSSION TITLE
Model Card for SPOC

### DISCUSSION THREAD

**Discussion by jermnelson** [sul-dlss-labs/spoc#31] [Ideas]
At today's AI-LAM Metadata working group meeting, we partly talked about using [Model Cards](https://arxiv.org/pdf/1810.03993.pdf) as a means to improve transparency in describing the models being used in machine learning projects. 

I think filling out the various sections contained within a Model Card would help us in general with this project. Interestingly, Google has a [Model Toolkit](https://ai.googleblog.com/2020/07/introducing-model-card-toolkit-for.html) for help in generating model cards. If people agree, I can add an issue ticket on create a Model Card for SPOC.

Thoughts?

**Comment by amandawhitmire** [comment]
Sounds like a great idea, @jermnelson! I'll read up and see where I can help.
